src/lib/adapters/transaction.ts

```diff
@@ -97,26 +95,23 @@ const gnosisBlockchain: BlockchainNetwork = {
 	},
 }
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const sepoliaBlockchain: BlockchainNetwork = {
-	name: 'Sepolia testnet',
-	provider: 'https://rpc2.sepolia.org/ ',
-	explorer: {
-		name: 'Etherscan',
-		url: 'https://sepolia.etherscan.io/',
-	},
-	nativeToken: {
-		name: 'Sepolia ETH',
-		symbol: 'SEP',
-		decimals: 18,
-		image: 'https://s2.coinmarketcap.com/static/img/coins/64x64/1027.png',
-	},
-	objects: {
-		splitterFactory: '0x0',
-	},
+let defaultBlockchainNetwork: BlockchainNetwork
+switch (PUBLIC_BLOCKCHAIN) {
```

Comment:
Stylistically I would prefer this to be a function that returns a value which gets assigned to the `defaultBlockchainNetwork` variable, therefore eliminate mutability, but this is also fine.

Reply:
Nice!